yarn.lock

```diff
@@ -2575,8 +2637,8 @@ ethereum-client-binaries@ethereum/ethereum-client-binaries:
     lodash.isempty "^4.4.0"
     lodash.values "^4.3.0"
     mkdirp "^0.5.1"
-    node-unzip-2 "github:glebdmitriew/node-unzip-2"
     tmp "0.0.29"
+    unzip "github:glebdmitriew/node-unzip-2"
```

Comment:
Got this error from travis.
```
[22:24:19] Using gulpfile ~/build/ethereum/mist/gulpfile.js
[22:24:19] Starting 'test'...
module.js:549
    throw err;
    ^
Error: Cannot find module 'node-unzip-2'
```

Reply:
hm it is in `devDependencies`. i am getting `Unresolved node modules: node-unzip-2` sometimes though with electron-builder so i'll try moving it to `dependencies` and see if that fixes